handlers/search.go

```diff
@@ -267,7 +265,42 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 				WHERE from_address_text LIKE $1 || '%'
 			) a 
 			GROUP BY from_address_text`, lowerStrippedSearch)
-
+	case "count_indexed_validators_by_withdrawal_credential":
+		var ensData *types.EnsDomainResponse
+		if utils.IsValidEnsDomain(search) {
+			ensData, _ = GetEnsDomain(search)
+			if len(ensData.Address) > 0 {
+				lowerStrippedSearch = strings.ToLower(strings.Replace(ensData.Address, "0x", "", -1))
+			}
+		}
+		if !searchLikeRE.MatchString(lowerStrippedSearch) {
```

Comment:
**minor:** `searchLikeRE` allows for up to 96 characters (pub keys & BLS) but withdrawal credentials only have 64 (like tx).
A more restrictive RE would avoid unnecessary queries.

Edit: As I asked in the BIDS we want to restrict the search to exact withdrawal credentials so the `utils` function `IsValidWithdrawalCredentials` should be used.

Reply:
done.
Thanks a lot for informing me that `utils.IsValidWithdrawalCredentials()` exists.